src/main/java/com/ericsson/ei/config/ConfigurationValidator.java

```diff
@@ -31,9 +31,10 @@ private void checkLoadRulesFile() {
         try {
             new RulesHandler().readRuleFileContent(rulePath);
         } catch (Exception e) {
-            LOGGER.debug("Rules file failed to be loaded/read. RuleFile path: " +  rulePath, e.getMessage());
+            LOGGER.debug("Rules file failed to be loaded/read. Path: " + rulePath, e.getMessage());
```

Comment:
We need to log the stacktrace. e.getMessage is only the message not the entire stacktrace.

Reply:
Done.